xplique/metrics/fidelity.py

```diff
@@ -401,7 +401,7 @@ def __init__(self,
                  metric: str = "loss",
                  batch_size: Optional[int] = 64,
                  causal_mode: str = "deletion",
-                 baseline_mode: Union[float, str] = 0.0,
+                 baseline_mode: Union[float, Callable] = 0.0,
```

Comment:
So we remove all the baselines mode @AntoninPoche have developed ? Like the negative / inverse / zero ?

Reply:
I spoke to Antonin, the best way is to have a Callable, one can develop its baseline_mode method
The code is changed in function of the type of the baseline_mode